Added logs.mdx file about log rotation by a docker daemon (#675)
getsentry/develop: src/docs/self-hosted/logs.mdx

---

> title: "Logs rotation"

Reviewer: ```suggestion
title: "Log rotation"
```

---

> By default docker writes logs of a container to one log file which can be giant - tens of gigabytes.

Reviewer: ```suggestion
By default docker writes logs of each container into a single file which can be giant - tens of gigabytes.
```

> To reduce a used space for logs you can 

Reviewer: ```suggestion
To reduce used disk space of these logs you can 
```

>  * modify docker-compose.yaml file by adding into a common section for each container following logging settings:

Reviewer: We use `.yml` extension in self-hosted, right now, I'm changing it just for convenience.
```suggestion
 * modify docker-compose.yml file and add following logging settings into a common section for each container:
```

Author: Please see next comment

```

> x-restart-policy: &restart_policy

Reviewer: This documentation is about log policy, not changing restart policies.
```suggestion
x-log-policy: &log_policy
```

Author: My original proposal to solve that problem was with minimal actions because “&restart_policy” is already used by each container.
If we're creating a new common section, we should reformulate following phrase:

    modify docker-compose.yml file and add following common logging setting and apply it either for each container or only for containers with huge log files:

Reviewer: It's not clear enough to me this way, also it will affect all containers as maybe users only want to apply this policy to some containers.

>   restart: unless-stopped

Reviewer: I think it would be better not to add this here.
```suggestion
```

  logging:
    driver: "json-file"
    options:
      max-size: "20m"
      max-file: "10"
```

> And docker daemon will automatically control size of a log file for each container from the yaml file: 20Mb * 10 = 200Mb for each container. <a href="https://docs.docker.com/compose/compose-file/compose-file-v3/#logging">Offical documentaion</a> about those settings.

Reviewer: ```suggestion
And docker daemon will automatically manage disk space used of each container's log file from the yaml file: 20Mb * 10 = 200Mb for each container. You can take a look at <a href="https://docs.docker.com/compose/compose-file/compose-file-v3/#logging">Offical documentaion</a> about those settings.
```

>  * modify a docker daemon <a href="https://docs.docker.com/config/containers/logging/configure/"><code>log settings</code></a> to apply them to all newly created containers.

Reviewer: ```suggestion
 * modify docker daemon <a href="https://docs.docker.com/config/containers/logging/configure/"><code>log settings</code></a> to apply them to all newly created containers system-wide.
```
